docs/unified/protectli/overview.md

```diff
@@ -0,0 +1,248 @@
+# Overview
+
+Select Model:
+
+=== "FW6"
+
+    * [Releases](
+        ../../variants/
+        protectli_fw6/releases.md) - list of all releases
+    * [Building manual](
+        ../../variants/protectli_fw6/building-manual.md) - how
+        to build Dasharo for Protectli FW6
+    * [Installation manual](
+        ../../variants/protectli_fw6/installation-\manual.md) -
+        how to install Dasharo for Protectli FW6
+    * [Hardware configuration matrix](
+        ../../variants/protectli_fw6/hardware-matrix.md) -
+        definition of hardware
+        configuration used during validation
+    * [Test matrix](
+        ../../variants/protectli_fw6/test-matrix.md) - validation
+        scope for the Protectli FW6
+
+=== "V1000 series"
+    The Vault is a small form network appliance built for use as a firewall /
+    router, virtualization platform, a daily-driven personal computer, and more.
+    The V1000 series feature an Intel Jasper Lake CPU, soldered down LPDDR4X
+    memory, Intel i225-V/i226-V 2.5G Ethernet ports, 4x USB-A 3.x ports, eMMC,
+    PCIe NVMe storage, M.2 WIFI and WWAN slots, serial console over USB-C and
+    a firmware TPM.
+
+    * V1210 - Intel® Celeron® N5105, 2x i225/i226, 4GB LPDDR4X
+    * V1211 - Intel® Celeron® N5105, 2x i225/i226, 8GB LPDDR4X
+    * V1410 - Intel® Celeron® N5105, 4x i225/i226, 8GB LPDDR4X
+    * V1610 - Intel® Pentium® Silver N6005, 6x i225/i226, 16GB LPDDR4X
+
+    ## Documentation sections
+
+    * [Releases](
+        ../../variants/protectli_v1000-series/releases.md)
+      - groups information about all releases.
+    * [Building manual](
+        ../../variants/protectli_v1000-series/building-manual.md)
+        - describes how to build Dasharo for Protectli V1000 series.
+    * [Initial deployment](
+        ../../variants/protectli_v1000-series/initial-deployment.md)
+        - describes initial Dasharo deployment methods
+        (i. e. flashing new firmware) for Protectli V1000 series.
+    * [Firmware update](
+        ../../variants/protectli_v1000-series/firmware-update.md)
+        - explains supported Dasharo open-source firmware update methods.
+    * [Recovery](
+        ../../variants/protectli_v1000-series/recovery.md)
+        - gathers information on how to recover the platform from
+        potential failure.
+    * [Hardware configuration matrix](
+        ../../variants/protectli_v1000-series/hardware-matrix.md)
+        - describes the platform's hardware configuration used during the
+        Dasharo firmware validation procedure.
+    * [Test matrix](
+        ../../variants/protectli_v1000-series/test-matrix.md)
+        - describes validation scope used during
+        Dasharo firmware validation procedure.
```

Comment:
All links lead to old non-unified files. Please fix for all Documenation sections in this file

Reply:
https://github.com/Dasharo/docs/pull/851/commits/0cb13f980e112b481f6cba2636cf6e3805c23fe7